api/mesh/options.proto

```diff
@@ -26,6 +26,19 @@ message KumaResourceOptions {
   bool skip_registration = 6;
 }
 
+message KumaWsOptions {
+  // ResourceName is the name of the policy for resource name usage in path
```

Comment:
Nit; double -check the field names in these comments, and end sentences with a full stop.